README.md

```diff
@@ -64,6 +64,10 @@ macros and more all the time (like canonical reformat using `grol -format` and w
 
 automatic memoization
 
+easy extensions/adding go functions to grol (see [extensions/extension.go](extensions/extension.go) for a lot of `math` additions)
+
+variadic functions both go side and grol side (using `..` on grol side)
```

Comment:
mostly because of the lexer which currently never needs to look at more than 2 characters at a time, also it's different in how go name's the "..." (though maybe both are bad reason and should use 3 dots anyway)

Reply:
I'm fine with it. I just asked you to document it as a comment